### PR TITLE
Closing #46 - Minor updates

### PR DESCRIPTION
**Pull request by Kalyan-Lama**
* Formatting
* Update images

### CODE REVIEW

File: source/api.md [hunk 1]
```diff
@@ -6,11 +6,28 @@ KoBoToolbox has two APIs for its primary tools, kpi and kc. Originally kc was th
...
+
+**Method 2:**
+
+* Select ACCOUNT SETTINGS (from where you normally log out) as shown in the image below:
+
+   ![image](/images/api/token1.png)
+
+* Then press the eye like button to view your Token as shown in the image below:
+
+   ![image](/images/api/token2.png)
```
Comment (joshuaberetta) [inline source/api.md:27]: Seems these two images haven't been checked into the repo

File: source/api.md [hunk 2]
```diff
@@ -6,11 +6,28 @@ KoBoToolbox has two APIs for its primary tools, kpi and kc. Originally kc was th
 
 The base URL depends on the server you are using: for most users it is [kf.kobotoolbox.org](https://kf.kobotoolbox.org) or [kobo.humanitarianresponse.info](https://kobo.humanitarianresponse.info). Below we only use [kpi-url] to refer to this base URL.
 
-**API Token:**
+### Getting your API Token:
```
Comment (joshuaberetta) [inline source/api.md:9]: You could add a third method if you like doing the following using `curl`:

```bash
curl -u username:password "https:/[kpi-url]/token/?format=json"
```

File: source/api.md [hunk 3]
```diff
@@ -6,11 +6,28 @@ KoBoToolbox has two APIs for its primary tools, kpi and kc. Originally kc was th
...
+
+* Select ACCOUNT SETTINGS (from where you normally log out) as shown in the image below:
+
+   ![image](/images/api/token1.png)
+
+* Then press the eye like button to view your Token as shown in the image below:
+
+   ![image](/images/api/token2.png)
+
+### Testing your API:
```
Comment (joshuaberetta) [inline source/api.md:29]: Make sure to put code within back ticks:

```bash
curl -X GET https://[kpi-url]/api/v2/assets.json -H "Authorization: Token [your_token_goes_here]"
```

**Review by joshuaberetta — CHANGES_REQUESTED** [review]

File: source/api.md (hunk 1 above)
Comment (joshuaberetta) [reply source/api.md:27]: Just saw that you added them in the other branch 😄

**Comment by Kalyan-Lama** [comment]
Thank you, @joshuaberetta. Your advice was constructive. Have updated as advised. Maybe we could now merge this with the master branch.

**Pull request merged**